src/components/Keycodes.vue

```diff
@@ -145,6 +185,17 @@ export default {
   cursor: pointer;
   margin-bottom: -1px;
 }
+.end-tab {
```

Comment:
Makes it look like there's a space between the tab name and the result count.

```suggestion
.tab span {
  margin-left: 4px;
}
.end-tab {
```